.github/codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 0%
+        base: auto
+        flags:
+          - unit
+#        paths:
+#          - "src"
+        # advanced settings
+        branches:
+          - master
+        if_ci_failed: ignore #success, failure, error, ignore
```

Comment:
does this mean it won't fail the job if CI fails?

Reply:
Yep, but since the ci is required it shouldnt allow a merge anyway